src/applications/personalization/profile-2/components/MHVTermsAndConditionsStatus.jsx

```diff
@@ -11,12 +10,12 @@ const MHVTermsAndConditionsStatus = ({ mhvAccount }) => {
 
   if (mhvAccount.termsAndConditionsAccepted) {
     return (
-      <>
+      <div className="vads-u-display--flex vads-u-flex-direction--column">
         <Verified>
           You’ve accepted the terms and conditions for using VA.gov health
           tools.
         </Verified>
-        <p className="vads-u-margin-bottom--0">
+        <p className="vads-u-margin-bottom--0 vads-u-padding-left--3 medium-screen:vads-u-padding-left--5">
```

Comment:
Needing this additional padding declared here makes me think that there's got to be some way to make the `ProfileInfoTable` better accommodate the extra padding for the rows that live next alongside rows that contain `Verified` components. But I'm not sure what that would be. Maybe adding the additional left padding for all the "values" and then the `Verified` component would have negative left padding?

Reply:
Hmm, I am not entirely sure what you mean.. `Verified` is not actually rendered in a different column, the only difference between tables with and without Verified components is some adding styling (padding and margin). We just render `row.value` in ProfileInfoTable, and I think adding the extra left padding in each separate section of the table (here `MHVTermsAndConditions` would be the most flexible solution -- moving the styling logic to ProfileInfoTable would make things more complicated in my opinion - would this be a blocker for you?

Reply:
Nope not a blocker at all!